docs/source-app/glossary/storage/drive_content.rst

```diff
@@ -136,9 +136,8 @@ Here is an illustrated code example on how to create drives within Works.
 
 .. code-block:: python
 
-    from lightning_app import LightningFlow, LightningWork
-    from lightning_app.core.app import LightningApp
-    from lightning_app.storage.drive import Drive
+    from lightning_app import LightningFlow, LightningWork, LightningApp
+    from lightning_app.storage import Drive
```

Comment:
should it be 

```suggestion
    from lightning import LightningFlow, LightningWork, LightningApp
    from lightning.storage import Drive
```

Reply:
I see the imports in the doc-source are not consistent either from `lightning_app` or `lightning`.  this can be addressed later.